.gitignore

```diff
@@ -7,3 +7,6 @@ tmp/
 
 # Local launch settings for .NET apps
 launchSettings.json
+
```

Comment:
Shouldn't need this any more as it's covered by `tmp/` on line 3, unless I'm mistaken about what the script does.

Reply:
Oops, yes.